### PR TITLE
fribidi@1.0.5: new package

### PR DESCRIPTION
**Pull request by goxberry**

### CODE REVIEW

**Comment by goxberry** [comment]
This package could be a `MesonPackage`, but the `strip` step seems to fail.

File: var/spack/repos/builtin/packages/fribidi/package.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+    depends_on('m4', type='build')
+    depends_on('autoconf', type='build')
+    depends_on('automake', type='build')
+    depends_on('libtool', type='build')
+
+    def url_for_version(self, version):
+        url  = 'https://github.com/fribidi/fribidi/releases/download/'
+        url += 'v{0}/fribidi-{0}.tar.bz2'
+        return url.format(version)
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/fribidi/package.py]: This package doesn't actually need a `url_for_version`, Spack is smart enough to figure it out:
```console
$ spack url parse https://github.com/fribidi/fribidi/releases/download/v1.0.5/fribidi-1.0.5.tar.bz2
==> Parsing URL: https://github.com/fribidi/fribidi/releases/download/v1.0.5/fribidi-1.0.5.tar.bz2

==> Matched version regex  0: r'^[a-zA-Z+._-]+[._-]v?(\\d[\\d._-]*)$'
==> Matched  name   regex  0: r'github\\.com/[^/]+/([^/]+)'

==> Detected:
    https://github.com/fribidi/fribidi/releases/download/v1.0.5/fribidi-1.0.5.tar.bz2
                               -------                                  ~~~~~
    name:    fribidi
    version: 1.0.5

==> Substituting version 9.9.9b:
    https://github.com/fribidi/fribidi/releases/download/v9.9.9b/fribidi-9.9.9b.tar.bz2
                               -------                                   ~~~~~~
```

Comment (goxberry) [reply var/spack/repos/builtin/packages/fribidi/package.py]: Removed `url_for_version` in https://github.com/spack/spack/pull/12386/commits/91e7ea41fb28d55af3b20060a0c12bc482107879

**Comment by goxberry** [comment]
Rebased on `develop`.

File: var/spack/repos/builtin/packages/fribidi/package.py [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    url      = "https://github.com/fribidi/fribidi/releases/download/v1.0.5/fribidi-1.0.5.tar.bz2"
+
+    version('1.0.5', sha256='6a64f2a687f5c4f203a46fa659f43dd43d1f8b845df8d723107e8a7e6158e4ce')
+
+    depends_on('m4', type='build')
+    depends_on('autoconf', type='build')
+    depends_on('automake', type='build')
+    depends_on('libtool', type='build')
+
+    # TODO: If https://github.com/spack/spack/pull/12344 is merged, this
```
Comment (scheibelp) [inline var/spack/repos/builtin/packages/fribidi/package.py]: @goxberry now that #12344 has been merged, you can remove this

Comment (goxberry) [reply var/spack/repos/builtin/packages/fribidi/package.py]: @scheibelp Done in https://github.com/spack/spack/commit/2ae0268043b257cf6e9bc02e9ff145da5cdca453, and rebased on `develop`.

**Pull request merged**